lib/sarif/bundle_audit_sarif.rb

```diff
@@ -50,5 +51,12 @@ def sarif_level(severity)
         SARIF_WARNINGS[:error]
       end
     end
+
+    def self.snippet_possibly_in_git_diff?(snippet, lines_added)
```

Comment:
Is the only used by specs? If so can we make this an rspe helper?

Reply:
No. This code is used outside specs.
In fact, `docker run ... --git-diff` calls this method.